src/test/java/org/benetech/servicenet/adapter/icarol/ICarolDataMapperTest.java

```diff
@@ -0,0 +1,318 @@
+package org.benetech.servicenet.adapter.icarol;
+
+import org.benetech.servicenet.adapter.icarol.model.ICarolAccessibility;
+import org.benetech.servicenet.adapter.icarol.model.ICarolAgency;
+import org.benetech.servicenet.adapter.icarol.model.ICarolContact;
+import org.benetech.servicenet.adapter.icarol.model.ICarolContactDetails;
+import org.benetech.servicenet.adapter.icarol.model.ICarolDay;
+import org.benetech.servicenet.adapter.icarol.model.ICarolHours;
+import org.benetech.servicenet.adapter.icarol.model.ICarolName;
+import org.benetech.servicenet.adapter.icarol.model.ICarolProgram;
+import org.benetech.servicenet.adapter.icarol.model.ICarolSite;
+import org.benetech.servicenet.domain.AccessibilityForDisabilities;
+import org.benetech.servicenet.domain.Eligibility;
+import org.benetech.servicenet.domain.Language;
+import org.benetech.servicenet.domain.Location;
+import org.benetech.servicenet.domain.OpeningHours;
+import org.benetech.servicenet.domain.Organization;
+import org.benetech.servicenet.domain.Phone;
+import org.benetech.servicenet.domain.PhysicalAddress;
+import org.benetech.servicenet.domain.PostalAddress;
+import org.benetech.servicenet.domain.Service;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Optional;
+import java.util.Set;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+public class ICarolDataMapperTest {
+
+    private ICarolDataMapper mapper;
+    private String PROVIDER_NAME = "provider name";
+
+    private static final String ID = "id";
```

Comment:
remove empty line

Reply:
It's intentional to separate static fields